dev-util/weka/weka-3.8.6-r1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit desktop java-pkg-2 java-pkg-simple xdg
+
+DESCRIPTION="A Java data mining package"
+HOMEPAGE="https://ml.cms.waikato.ac.nz/weka"
+SRC_URI="https://downloads.sourceforge.net/project/weka/weka-3-8/${PV}/weka-${PV//./-}.zip"
+S="${WORKDIR}/${P//./-}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="app-arch/unzip"
+CP_DEPEND="dev-java/javacup:0"
+DEPEND="${CP_DEPEND}
+	virtual/jdk:1.8
+	dev-java/commons-compress
```

Comment:
glad that you already found most of them. not so obvious is [MAVEN_ID="jakarta.xml.bind:jakarta.xml.bind-api:2.3.3" 
](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/jaxb-api/jaxb-api-2.3.3-r2.ebuild#n7)

did you also get compilation errors? asking because you added them to `DEPEND` while pure runtime deps might better go to `RDEPEND`

and, for java packages please never forget the slot. `dev-java/commons-compress:0`

Reply:
> glad that you already found most of them. not so obvious is [MAVEN_ID="jakarta.xml.bind:jakarta.xml.bind-api:2.3.3" ](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/jaxb-api/jaxb-api-2.3.3-r2.ebuild#n7)
> 

Thanks :)

> did you also get compilation errors? asking because you added them to `DEPEND` while pure runtime deps might better go to `RDEPEND`
> 
> and, for java packages please never forget the slot. `dev-java/commons-compress:0`

I didn't get any compilation errors. Yha `RDEPEND` is the correct place for them.